project/Dependencies.scala

```diff
@@ -44,8 +45,8 @@ object Dependencies {
 
   val asyncHttpClient = Seq("org.asynchttpclient" % "async-http-client" % "2.12.3")
 
-  val akkaStreams = Seq("com.typesafe.akka" %% "akka-stream" % "2.6.15")
-  val akkaHttp    = Seq("com.typesafe.akka" %% "akka-http" % "10.2.6")
+  val akkaStreams = Seq("com.typesafe.akka" %% "akka-stream" % "2.6.15" cross CrossVersion.for3Use2_13)
+  val akkaHttp    = Seq("com.typesafe.akka" %% "akka-http" % "10.2.6" cross CrossVersion.for3Use2_13)
```

Comment:
I think akka artifats should soon be published for Scala 3: https://github.com/akka/akka/issues/30243

Reply:
- TODO: await 2.6.18 https://github.com/akka/akka/pull/30817

Reply:
Seems ok now?